reshenie.java

```diff
@@ -0,0 +1,11 @@
+public class Main {
+    public static void main(String[] args) {
+        ball ball1 = Lottery.getRandomBall();
+        ball ball2 = Lottery.getRandomBall();
+        ball ball3 = Lottery.getRandomBall();
+        System.out.println(ball1);
+        System.out.println(ball2);
+        System.out.println(ball3);
+    }
+}
+}

```

Comment:
There is an extra closing brace `}` which is not needed and should be removed to avoid syntax errors.